sql-queries/sql-column-name/sql-column-name-mysql.sql

```diff
@@ -0,0 +1,7 @@
+SHOW COLUMNS FROM Course;
```

Comment:
I don't think this is needed for this tutorial, can you remove it from the PR, please?